src/components/DigitalCredentials/RegisterWallet.vue

```diff
@@ -19,6 +19,7 @@
             <v-col cols="12" lg="3" class="px-2">
               <!-- Placeholder Url -->
               <QrcodeVue
+                render-as="svg"
```

Comment:
Please attach a screenshot showing that this does what you intend.

Reply:
Old:
![image](https://user-images.githubusercontent.com/64869009/193946939-28f2fffd-02b9-46cd-8a76-689e3efec9ff.png)

New:
![image](https://user-images.githubusercontent.com/64869009/193946983-ff7a09b5-f93d-4933-ae9a-01fd3a3e4acb.png)

Reply:
Oh, that's an awesome improvement! 👍